engine/src/CalciteInterpreter.cpp

```diff
@@ -887,10 +973,10 @@ blazing_frame evaluate_query(
 						new_strings = NVStrings::create_from_array(nullptr, 0);
```

Comment:
Can you get rid of the NVStrings stuff? We dont need it anymore

Reply:
was deleted